services/makecode-extensions.json

```diff
@@ -8,15 +8,6 @@
       "default": "modules.accelerometer"
     }
   },
-  {
```

Comment:
this is wrong

Reply:
This file is built automitically from scanning the linked repo pxt-jacdac. Looks like you need to pull pxt-jacdac#master.